WooCommerce/Classes/POS/Presentation/Orders/PointOfSaleOrderDetailsView.swift

```diff
@@ -0,0 +1,62 @@
+import SwiftUI
+
+@available(iOS 17.0, *)
```

Comment:
We no longer need to mark these with iOS17 availability, it can be removed across all files 👍